### ISSUE TITLE
MMU: INSTR_ACCESS_FAULT triggered by PMP

### ISSUE THREAD

**Issue by masgia** [openhwgroup/cva6#606]
In mmu.sv, line 247, there's a mixture of vaddr and PLEN, which causes a misalignment in `tval`
```icache_areq_o.fetch_exception = {riscv::INSTR_ACCESS_FAULT, {{riscv::XLEN-riscv::PLEN{1'b0}}, icache_areq_i.fetch_vaddr}, 1'b1};```

**Comment by MikeOpenHWGroup** [comment]
Hi @zarubaf and @JeanRochCoulon.  This is a report of a possible RTL bug that nobody has looked at.  The code snip-it above is still in the CVA6 code.  Please have a look.

**Comment by Jbalkind** [comment]
I think this was fixed or at least made consistent in #968

**Comment by JeanRochCoulon** [comment]
Indeed, it is solved !   Thanks for the fix !  @masgia to my mind you can close the issue

**Comment by cainria** [comment]
This issue can be closed
